pom.xml

```diff
@@ -66,7 +66,7 @@
       <dependency>
         <groupId>io.vertx</groupId>
         <artifactId>vertx-stack-depchain</artifactId>
-        <version>3.8.5</version>
+        <version>3.9.0</version>
```

Comment:
Can you also update the vertx version in sample/pom.xml and sample2/pom.xml?
Can I remove the other RMB-619 branch mentioned on Slack and in the Development section on https://issues.folio.org/browse/RMB-619?

Reply:
Updated, sample and sample2.. Funny, mvn versions:set  does not set it (it was also wrong for the 29.5.0-SNAPSHOT, where sample, sample2 was pointing to an older version).

Perhaps it's time to retire sample and sample2?

I'm confused what branch you are talking about.

Reply:
The pom.xml of sample and sample2 contains `<vertx-version>3.8.5</vertx-version>`, this should be updated.

Reply:
sample can be removed.
sample2 servers as a complete integration test. If we can integrate this into domain-models-runtime we can remove sample2 as well.

Reply:
I talk about this branch: https://github.com/folio-org/raml-module-builder/compare/RMB-619-vert.x-3.9.0

Reply:
Done

Reply:
OK. It took a while to understand. Need a break. So we both did the same work?